sourcegraph/sourcegraph
# Code Insights: Update visx packages and remove legacy unused chart packages

Closes https://github.com/sourcegraph/sourcegraph/issues/33712

## Test plan
- Make sure that all snapshots and code insights UI (the main consumer of visx charts components) didn't get any regressions - visual and logical 
- Make sure that we don't have any visx packages which we don't use in our package.json file



## App preview:

- [Web](https://sg-web-vk-update-chart-dependecies.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-hgwcadypqo.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
